.travis-xs-opam.sh

```diff
@@ -4,10 +4,10 @@ set -ex
 
 PACKAGE="xapi"
 PINS="xapi:. xapi-cli-protocol:. xapi-client:. xapi-consts:. xapi-database:. xapi-datamodel:. xapi-types:. xe:."
-BASE_REMOTE="https://github.com/xapi-project/xs-opam.git"
+BASE_REMOTE="https://github.com/xapi-project/xs-opam.git#quebec"
```

Comment:
This is already defined in the xs-opam-ci.env. the variables for setting the remote and its branch are different ones, looking at it.